### PR TITLE
Test DHCP from Corsham

### PR DESCRIPTION
**Pull request by emileswarts**
This creates an instance in production that is used as a jump box to get onto the VM in Corsham.

Once on the VM, tests can be run using perfDHCP on the production DHCP load balancers.
Documented in the README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -54,3 +54,35 @@ The code for building this container can be found [here](https://github.com/mini
...
+## Corsham Test site
+
+We do remote testing of the DHCP service from a virtual machine running in Corsham.
+To access this VM you need to go through the bastion set up in production, which is on the same network (via the Transit Gateway) as the VM.
+
+### Bastion details:
+
+SSH key - `/corsham/testing/bastion/private_key`
+
+Once on the Bastion run:
```
Comment (jivdhaliwal) [inline README.md:67]: How do you get on the bastion? Is that outside the scope of this readme?

File: README.md [hunk 2]
```diff
@@ -54,3 +54,35 @@ The code for building this container can be found [here](https://github.com/mini
...
 [Azure AD manual provisioning guide](docs/azure_ad.md)
+
+## Corsham Test site
+
+We do remote testing of the DHCP service from a virtual machine running in Corsham.
+To access this VM you need to go through the bastion set up in production, which is on the same network (via the Transit Gateway) as the VM.
+
+### Bastion details:
+
+SSH key - `/corsham/testing/bastion/private_key`
```
Comment (jivdhaliwal) [inline README.md:65]: Is this the location of the key on the 'bastion' ? And do we need to point to it when we ssh in to the VM ?

File: README.md [hunk 3]
```diff
@@ -54,3 +54,35 @@ The code for building this container can be found [here](https://github.com/mini
...
+Once on the Bastion run:
+
+```bash
+ssh root@<VM_IP_ADDRESS>
+```
+
+### VM details:
+
+IP address - `/corsham/testing/vm/ip_address`
+Password - `/corsham/testing/vm/password`
```
Comment (jivdhaliwal) [inline README.md:76]: When would we need the ip address or password?

File: README.md (hunk 1 above)
Comment (emileswarts) [reply README.md:67]: You SSH onto the bastion

File: README.md (hunk 2 above)
Comment (emileswarts) [reply README.md:67]: These are parameters in production SSM, added a sentence to mention this above.

File: README.md (hunk 1 above)
Comment (jivdhaliwal) [reply README.md:69]: But how? Do you have to find all the info in the aws console? The terraform output? Follow this guide https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AccessingInstancesLinux.html ?

**Review by jivdhaliwal — APPROVED** [review]
Thanks for making the changes :shipit:

**Pull request merged**